tests/unit/test_connection_name.py

```diff
@@ -14,20 +14,37 @@
 
 import pytest  # noqa F401 Needed to run the tests
 
-from google.cloud.sql.connector.connection_name import _parse_instance_connection_name
+# fmt: off
```

Comment:
import style of long import names differs between `isort` and `black` format, turning black formatter off for these imports to stop lint job from complaining.